POEApi.Model/Item.cs

```diff
@@ -53,6 +54,7 @@ public abstract class Item : ICloneable
         public string TradeInventoryId { get; set; }
         public string Character { get; set; }
         public int ItemLevel { get; set; }
+        public string ArtFilename { get; set; }
```

Comment:
ArtFilename is a consideration only of divination cards - this should be in DivinationCard.cs instead as it will be null for everything else.